hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ApiPermissionConfig.java

```diff
@@ -247,6 +249,7 @@ public record ApiPermissionConfig(
         @ConfigProperty(defaultValue = "2-58") PermissionedAccountsRange freeze,
         @ConfigProperty(defaultValue = "2-50") PermissionedAccountsRange getAccountDetails,
         @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenUpdateNfts,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenReject,
```

Comment:
nit : Not a blocker on this PR. Should we also add `tokenAirdrop` here ? Seems like adding `tokenReject` in pendingState seems different.

Reply:
We add this property because we getting our protos from a branch that has the token reject functionality as well. If the token reject PR is merged first we won't have these changes, as of now they are here so the code compiles. As for the token Airdrop it will be added with different PR that introduces the creation of the Airdrops, as this PR covers only the state we don't need it for now.

Reply:
Added token Airdrop as well, tokenReject is here because we are using a proto that has the changes for token reject as well, just so that the code compiles.